src/utils/functionRuntimeUtils.ts

```diff
@@ -82,26 +103,22 @@ export namespace functionRuntimeUtils {
     }
 
     async function getLocalFunctionRuntimeVersion(): Promise<string | null> {
-        const versionInfo: string = await cpUtils.executeCommand(undefined, undefined, 'npm', 'ls', runtimePackage, '-g');
-        const matchResult: RegExpMatchArray | null = versionInfo.match(/(?:.*)azure-functions-core-tools@(.*)/);
+        const versionInfo: string = await cpUtils.executeCommand(undefined, undefined, 'func');
+        const matchResult: RegExpMatchArray | null = versionInfo.match(/(?:.*)Azure Functions Core Tools (.*)/);
```

Comment:
We should use `func --version` and fall back to this logic only if it doesn't work. Since `func --version` isn't released yet, I say just file an issue on it for now